src/client/components/HealthCareApp.jsx

```diff
@@ -54,15 +54,20 @@ class HealthCareApp extends React.Component {
   getUrl(direction) {
     const routes = this.props.route.childRoutes;
     const panels = [];
+    const data = this.props.data;
     let currentPath = this.props.location.pathname;
     let nextPath = '';
 
-    // TODO(awong): remove the '/' alias for '/introduction' using history.replaceState()
```

Comment:
Doesn't look like we need to, especially since we're going to be porting all this over to vets-website.